.github/workflows/create_release.yml

```diff
@@ -11,8 +15,7 @@ permissions:
 jobs:
   create_tag_and_release:
     # The tag and the release should be created only when the automated PR created by `create_pr_for_changelog` is merged
-    # Or to test it for now, by triggering a workflow dispatch by hand
-    if: (github.event.pull_request.merged == true && startsWith(github.event.pull_request.title, 'chore(release)') && contains(github.event.pull_request.labels.*.name, 'automated') && contains(github.event.pull_request.labels.*.name, 'pending-release'))
+    if: ((github.event.pull_request.merged == true && startsWith(github.event.pull_request.title, 'chore(release)') && contains(github.event.pull_request.labels.*.name, 'automated') && contains(github.event.pull_request.labels.*.name, 'pending-release'))|| github.event_name == 'workflow_dispatch')
```

Comment:
ah, too late but I suspect this could have been spread over multiple lines to increase legibility